inductiva/tasks/task.py

```diff
@@ -238,6 +239,8 @@ def __str__(self):
         table_str += f"\n{wall_time_table}"
         table_str += f"\nTime breakdown:\n{time_metrics_table}"
         table_str += f"\nData:\n{data_metrics_table}\n"
+        if self.cost:
```

Comment:
Should we include the same null check for the 'status' attribute?

Reply:
A task will always have a status.

Reply:
In theory, yes. But, the constructor of the 'TaskInfo' class sets the 'status' attribute to None. It's just a detail for consistency.

Reply:
I only added this check because a task will not have a cost until its finished. Without the "if" we would have cost: None until the task was finished.

For status we don't need to do the if because it will never be None, and we want to print the status every time. Adding an if would just make the code harder to read without any advantage.